logbook-api/src/main/java/org/zalando/logbook/HttpStatus.java

```diff
@@ -0,0 +1,100 @@
+package org.zalando.logbook;
+
+import lombok.Getter;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Getter
+public enum HttpStatus {
```

Comment:
I'd not make it public at the start, as we don't need this enum in other parts for now. Reverting this will be hard.

Reply:
Thank you for the suggestion!